rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -359,6 +360,13 @@ upgrade_ncf() {
 
 }
 
+# Upgrade masterfiles
+upgrade_masterfiles() {
+  # - 3.0.13  Now we should have a masterfiles initialized from initial promises
+  STEP="Upgrade masterfiles from initial promises"
+  rsync --delete -rptgoq "${RUDDER_SHARE}/initial-promises/" "${RUDDER_VAR}/cfengine-community/masterfiles/"
```

Comment:
I think we should add a ```mkdir -p "${RUDDER_VAR}/cfengine-community/masterfiles/"``` before the rsync, to make absolutely sure this can't cause an error on upgrade - especially if we include it in existing released branches.

It is *possible* that this can be executed before the agent is installed, therefore before that directory exists.